### PR TITLE
Ui implementation

### PR DESCRIPTION
**Pull request by Sassah-Wingman**
Added WindowUI Derived Class!

### CODE REVIEW

**Comment by matferib** [comment]
Sassah, I don't think you intend to merge this, do you?

**Comment by Sassah-Wingman** [comment]
I do.  I just deleted those files.   Delete them as well.

File: main.cpp [hunk 1]
```diff
@@ -2,6 +2,8 @@
 #include <iostream>
 #include <memory>
 #include <string>
+#include <wx/wx.h>
```
Comment (matferib) [inline main.cpp]: Please drop.

Comment (Sassah-Wingman) [reply main.cpp]: ok

File: main.cpp [hunk 2]
```diff
@@ -46,7 +47,14 @@ class CommandLineUI : public GenericUI {
   }
 };
 
-// TODO(sassah): implement a windows UI that inherits from GenericUI.
+// TODO(sassah): implement a windows UI (WindowUI) that inherits from GenericUI.
+    class WindowUI : public GenericUI {
```
Comment (matferib) [inline main.cpp]: Please follow alignment. No indent before class.

**Review by matferib — APPROVED** [review]

**Pull request merged**